tools/depends/native/TexturePacker/src/TexturePacker.cpp

```diff
@@ -118,6 +118,7 @@ void CreateSkeletonHeaderImpl(CXBTFWriter& xbtfWriter, std::string fullPath, std
         }
       }
     }
+    if (errno) fprintf(stderr, "Error reading directory %s (%s)\n", fullPath.c_str(), strerror(errno));
```

Comment:
Can you begin if clauses on a newline to match our code style?

Reply:
@garbear my bad, fixed.

Reply:
@garbear how do you like it now?

Reply:
looks good now, but I have no way of testing atm

Reply:
Fair enough. I guess we will let it rot for a bit until someone notices the smell and handles it? :nauseated_face: